recipes/openfx/all/test_package/invert.cpp

```diff
@@ -0,0 +1,299 @@
+/*
```

Comment:
test_package could be reduced, but it's fine

Reply:
Yes, i could be reduced. Don't need to be a functional example, only consume few methods to validate the linkage and packaged artifacts.